types/react-native-fbsdk/index.d.ts

```diff
@@ -0,0 +1,1089 @@
+// Type definitions for react-native-fbsdk 0.6
+// Project: https://github.com/facebook/react-native-fbsdk
+// Definitions by: Ifiok Jr. <https://github.com/ifiokjr>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { ComponentClass, Component } from 'react';
+import { StyleSheet } from 'react-native';
+
+// tslint:disable-next-line:export-just-namespace
```

Comment:
Why ignore this rule? You can fix this by removing the `declare namespace FBSDK` wrapper and exporting everything from the top-level.